src/07-projects/project-items.apib

```diff
@@ -120,4 +120,20 @@ Get details for a single project item.
                 + type: `product` (string)
                 + id: `e2314517-3cab-4aa9-8471-450e73449040` (string)
             + added_at: `2018-12-04T16:44:33+00:00` (string)
-            + updated_at: `2018-12-05T16:44:33+00:00` (string)
\ No newline at end of file
+            + updated_at: `2018-12-05T16:44:33+00:00` (string)
+
+
+### projectItems.updateInternalRate [POST /projectItems.updateInternalRate]
+
+Set participant user rates for timetracking project items, optionally for a single project
```

Comment:
Q: does this endpoint also update the user's global internal rate? I think we should document this.